dip-0004.md

```diff
@@ -165,17 +165,12 @@ The `MNLISTDIFF` message contains a serialized object with the following structu
 | deletedMNs | uint256[] | variable | A list of ProRegTx hashes for masternode which were deleted after baseBlockHash |
 | mnCount | compactSize uint | 1-9 | Number of SML entries which were added or updated since baseBlockHash |
 | mnList | SMLEntry[] | variable | The list of SML entries which were added or updated since baseBlockHash |
-
-Starting with protocol version 70214, the following fields are added to the `MNLISTDIFF` message.
-
-| Field | Type | Size | Description |
-| --- | --- | --- | --- |
 | deletedQuorumsCount | compactSize uint | 1-9 | Number of LLMQs which were deleted from the active set after baseBlockHash |
```

Comment:
@ogabrielides @PastaPastaPasta @UdjinM6 I like simplifying this into one table, but what do you think about adding a note that indicates when the fields were introduced? Helpful or unnecessary?
```suggestion
| deletedQuorumsCount | compactSize uint | 1-9 | Number of LLMQs which were deleted from the active set after baseBlockHash. _Added in protocol version 70214_ |
```

Reply:
@thephez Or maybe add an extra column saying the protocol version per field?

Reply:
@ogabrielides Yes, that could work too ([example](https://github.com/thephez/dips/blob/thephez-patch-1/dip-0004.md#requesting-the-masternode-lists)). Curious to see other opinions.